package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edie-processors",
-  "version": "0.0.10",
+  "version": "0.0.11",
   "description": "A set of helper methods",
   "main": "./lib/index.js",
   "files": [
```

Comment:
Not all the tests are running, I have this fix in another branch but you could add this to double check that all the tests run.
`"test": "node ./node_modules/mocha/bin/mocha --require @babel/polyfill --require @babel/register tests/**"`